### PR TITLE
Increases volume of the sound effect for knocking on firelocks

### PR DESCRIPTION
**Pull request by E231kei500bandai**
## About The Pull Request

Changed firelocks to have the same bash and knock sound effects as windows.
Which increases the volume of the knock sound effect and also adds the bash sound effect.

Also improves some of the code for firelock and window 

## Why It's Good For The Game

Firelocks already had the knock sound effect, but it is so quiet that you can barely hear it. The volume should be increased so that people can hear it.

## Changelog

:cl:
sound: adds bash sound effect to firelocks and increases volume of knock sound effect on firelocks
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/firedoor.dm [hunk 1]
```diff
@@ -25,6 +25,8 @@
 	var/boltslocked = TRUE
 	var/list/affecting_areas
 	var/being_held_open = FALSE
+	var/knocksound = 'sound/effects/Glassknock.ogg'
+	var/bashsound = 'sound/effects/Glassbash.ogg'
```
Comment (LemonInTheDark) [inline code/game/machinery/doors/firedoor.dm:29]: These sound effects don't exist. Does this actually work? (Insert comments about windows here)

File: code/game/machinery/doors/firedoor.dm [hunk 2]
```diff
@@ -82,17 +84,22 @@
...
-		span_notice("You bang on \the [src]."))
-	playsound(loc, 'sound/effects/glassknock.ogg', 10, FALSE, frequency = 32000)
+	if(!user.combat_mode)
+		user.visible_message(span_notice("[user] knocks on [src]."), \
+			span_notice("You knock on [src]."))
+		playsound(src, knocksound, 50, TRUE)
+	else
+		user.visible_message(span_warning("[user] bashes [src]!"), \
+			span_warning("You bash [src]!"))
+		playsound(src, bashsound, 100, TRUE)
```
Comment (LemonInTheDark) [inline code/game/machinery/doors/firedoor.dm:102]: Undocced change, sad

File: code/game/machinery/doors/firedoor.dm (hunk 1 above)
Comment (BlueMemesauce) [reply code/game/machinery/doors/firedoor.dm:29]: What do you mean they don't exist? The files are in the code.
https://github.com/tgstation/tgstation/blob/master/sound/effects/glassbash.ogg
https://github.com/tgstation/tgstation/blob/master/sound/effects/glassknock.ogg

I tested it in game and it works.

Comment (tf-4) [reply code/game/machinery/doors/firedoor.dm]:
```suggestion
	var/knock_sound = 'sound/effects/Glassknock.ogg'
	var/bash_sound = 'sound/effects/Glassbash.ogg'
```

File: code/game/machinery/doors/firedoor.dm [hunk 3]
```diff
@@ -82,17 +84,22 @@
...
 		return
 	user.changeNext_move(CLICK_CD_MELEE)
 
-	user.visible_message(span_notice("[user] bangs on \the [src]."), \
-		span_notice("You bang on \the [src]."))
-	playsound(loc, 'sound/effects/glassknock.ogg', 10, FALSE, frequency = 32000)
+	if(!user.combat_mode)
+		user.visible_message(span_notice("[user] knocks on [src]."), \
+			span_notice("You knock on [src]."))
+		playsound(src, knocksound, 50, TRUE)
```
Comment (tf-4) [inline code/game/machinery/doors/firedoor.dm:98]:
```suggestion
		playsound(src, knock_sound, 50, TRUE)
```

File: code/game/machinery/doors/firedoor.dm (hunk 2 above)
Comment (tf-4) [inline code/game/machinery/doors/firedoor.dm:102]:
```suggestion
		playsound(src, bash_sound, 100, TRUE)
```

**Comment by LemonInTheDark** [comment]
Bad bot

**Pull request merged**